nodejs/week1/avg.js

```diff
@@ -0,0 +1,22 @@
+console.log(getAverage(process.argv));
+
+/**
+ *
+ * @param {Array} arguments - list of arguments the user have entered. Presumes argument[0] and argument[1]
+ *  to always be system default peth-arguments, which are removed.
+ * @returns {Number} - the average of provided arguments (all non-number values are considered to be 0);
+ */
+function getAverage(arguments) {
+  arguments.splice(0, 2);
```

Comment:
'arguments' could be a reserved javascript variable so change it to another name
https://www.w3schools.com/js/js_reserved.asp

Reply:
Great suggestion! Totally forgot about it. Replaced the arguments with userArguments